README.md

```diff
@@ -1,2 +1,30 @@
-# vvgo
-Virtual Video Game Orchestra
+# Virtual Video Game Orchestra
+
+:wave: We are the Virtual Video Game Orchestra (VVGO for short), an orchestra organized by members from various IRL VGOs/GSOs, and comprised of local musicians hailing from across the globe!
+
+## Build
+
+You can build the webserver either using docker or go build tools. 
+When you run it, you can visit the site at http://localhost:8080.
+
+### Build with docker
+
+```
+# Clone the repo
+$ git clone https://github.com/virtual-vgo/vvgo.git
+# Build the docker image
+$ docker build --tag vvgo .
+# Start the container
+$ docker run -p8080:8080 --rm vvgo
+```
```

Comment:
Same as below.
```suggestion
```sh
# Clone the repo
git clone https://github.com/virtual-vgo/vvgo.git
# Build the docker image
docker build --tag vvgo .
# Start the container
docker run -p8080:8080 --rm vvgo
```
```